frontend/src/Search.elm

```diff
@@ -472,7 +473,7 @@ createUrl toRoute model =
 
 type Channel
     = Unstable
-    | Release_21_11
+    | Release String
```

Comment:
Why do we still have an `Unstable` constructor if the unstable channel is in Channels.elm?

Reply:
If it's `Unstable` then ChannelDetails are `ChannelDetails "unstable" "unstable" "nixos/trunk-combined" "nixos-unstable"` instead of `ChannelDetails rls rls ("nixos/release-" ++ rls) ("nixos-" ++ rls)`

Reply:
So we don't want the unstable channel in the channel list then?